src/extra/gfx/feature.cpp

```diff
@@ -293,9 +868,13 @@ struct FeatureShard {
 
     generatorInstanceData.inputType = GeneratedDrawInputTableType;
     _drawableGeneratorBranch.compose(generatorInstanceData);
+    // _drawableGeneratorsRaw = _drawableGenerators;
+    _drawableGeneratorBranch.setRunnables(_drawableGenerators);
 
     generatorInstanceData.inputType = GeneratedViewInputTableType;
     _viewGeneratorBranch.compose(generatorInstanceData);
+    // _viewGeneratorsRaw = _viewGenerators;
+    _viewGeneratorBranch.setRunnables(_viewGenerators);
```

Comment:
The current implementation would result in a format exception as the shard cannot resolve the output type of the generators.

Reply:
Call `setRunnables` in `composeGeneratorWire` results in the following error being thrown:
```
[trace] [2023-06-07 11:33:33.875] [T-142709] [foundation.hpp::434] Creating wire: <shader wire>
[error] [2023-06-07 11:33:33.876] [T-142709] [runtime.cpp::1091] Error composing shard: Failed to find shader parameter "time" in any buffer, wire: <shader wire>
```